routes/post.js

```diff
@@ -58,4 +57,54 @@ import { newPost } from "../controllers/post.js";
 
 router.post("/", newPost);
 
+/**
+ * @swagger
+ * /post/{postId}:
+ *  get:
+ *    tags:
+ *    - "post"
+ *    summary: Get post information (description, comments etc.)
+ *    security:
+ *    - bearerAuth: []
+ *    parameters:
+ *    - name: "userId"
+ *      in: "path"
+ *      description: "User ID"
+ *    responses:
+ *      '200':
+ *        description: A successful response, denoting that the post information has been successfully fetched
+ *        schema:
+ *          type: object
+ *          properties:
+ *            success:
+ *              type: boolean
+ *              default: true
+ *            data:
+ *              type: object
+ *              description: Post information
```

Comment:
It would be nice to document the structure of the `post` object. See this as an example: https://github.com/nibble-lumen/washere-backend/blob/main/routes/post.js#L22